homeassistant/components/netgear_lte/__init__.py

```diff
@@ -150,8 +150,10 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
 
     if lte_config := config.get(DOMAIN):
         for entry in lte_config:
-            result = await hass.config_entries.flow.async_init(
-                DOMAIN, context={"source": SOURCE_IMPORT}, data=entry
+            result = await hass.async_create_task(
```

Comment:
Does this really make a difference? If the block is due to config flow waiting for `async_setup`, I think the whole import and issue creation should be a separate task that is scheduled to complete after `async_setup`.

Reply:
Exactly how do you see that looking? Without the last commit, I have to keyboard interrupt and the logs indicate maximum recursion.

Reply:
Break out the YAML import to a new coroutine function that we can schedule as a task.
```py
async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
    ...
    hass.async_create_task(import_yaml(hass, config))
    return True


async def import_yaml(hass: HomeAssistant, config: ConfigType) -> None:
    if lte_config := config.get(DOMAIN):
        ...
```

Reply:
Done, small tweak to only do so if we have entry in yaml. Save some indent.